docs/section10/10-14.md

```diff
@@ -0,0 +1,189 @@
+                    - Check if efficiency_matches_acceptable_value equals False, if it equals True then skip below: `if efficiency_matches_acceptable_value == False:`  
+                        - Reset was_efficiency_found equal to FALSE: `was_efficiency_found = FALSE`
+                        - Loop through each of the efficiency_metric_types associated with the heating system until FULL_LOAD_EFFICIENCY_NO_FAN is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`
+                            - Check if the efficiency metric equals FULL_LOAD_EFFICIENCY_NO_FAN: `if heating_system_b.efficiency_metric_types[x] == "FULL_LOAD_EFFICIENCY_NO_FAN":` 
+                                - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`
+                                - Set the efficiency found boolean variable to TRUE: `was_efficiency_found = TRUE`
+                            - Else, do nothing and move on to the next metric: `Else:`   
+                        - Check if was_efficiency_found equals TRUE: `if was_efficiency_found == TRUE:`  
+                            - Check if the modeled efficiency meets the expected baseline efficiency by using the RCT compare_standard_val function, Permit_More_Stringent_Value is defined globally and is TRUE OR FALSE: `efficiency_matches_acceptable_value = compare_standard_val(Permit_More_Stringent_Value, expected_baseline_eff_b, modeled_efficiency_b, "greater than")`  func shall return True or False, "greater than" means that if the modeled_efficiency_b is greater than or equal to the expected_baseline_eff_b it would be accepted if Permit_More_Stringent_Value equals TRUE   
+            - Check if cooling capacity is >=65,000 Btuh and <135000 Btuh OR the zone aggregation factor was defined and the capacity is >=65,000 Btuh and <135000 Btuh OR the capacity was not defined (this is one of the most stringent) AND the efficiency matches acceptable value is equal to FALSE: `if [(zone_list_b[0].aggregation_factor != Null AND total_heating_capacity_b/zone_list_b[0].aggregation_factor >= 65000 AND total_heating_capacity_b/zone_list_b[0].aggregation_factor < 135000) OR (total_cooling_capacity_b >= 65000 AND total_cooling_capacity_b < 135000) OR was_cool_capacity_not_defined == FALSE OR (zone_list_b[0].aggregation_factor == Null AND total_heating_capacity_b >= 135000)] AND efficiency_matches_acceptable_value == FALSE:` 
```

Comment:
Agreed with respect to the was_cool_capacity_not_defined variable.

The reason for efficiency_matches_acceptable_value == FALSE is because if the previous if statement was triggered and the efficiency was found to meet an acceptable value then there is no reason to run this next set of logic.